AUTHORS.txt

```diff
@@ -1,4 +1,4 @@
-# ACE3 CONTRIBUTOR LIST
+﻿# ACE3 CONTRIBUTOR LIST
```

Comment:
please don't change bom mark.

Reply:
Think Atom did this automatically.
Fixed